client/my-sites/checkout/src/components/item-variation-picker/util.ts

```diff
@@ -14,6 +14,15 @@ export function getItemVariantCompareToPrice(
 		return undefined;
 	}
 
+	// Ignore a 1 month discount when calculating the discount percentage
+	if (
+		variant.termIntervalInMonths === 24 &&
```

Comment:
Small nitpick, this is the only place I see `termIntervalInMonths` used in the function. Not sure if it makes sense to make all the checks consistent where you check the interval and term? (I think this works fine, was just curious)

Reply:
I will merge this for now but I will look at improvements in a separate PR where I am adding a test